File: Backend_TFG/src/main/java/com/gert/tfgdam/entity/FinalizarCompra.java
package com.gert.tfgdam.entity;

import java.util.List;

public class FinalizarCompra {
    private Venta venta;
    private List<LineaVenta> lineasVenta;

    public Venta getVenta() {
        return venta;
    }

    public void setVenta(Venta venta) {
        this.venta = venta;
    }

    public List<LineaVenta> getLineasVenta() {
        return lineasVenta;
    }

    public void setLineasVenta(List<LineaVenta> lineasVenta) {
        this.lineasVenta = lineasVenta;
    }
}

File: Frontend_TFG/app/src/main/java/com/gert/tfgdam/feature/user/pago/model/FinalizarCompra.kt
package com.gert.tfgdam.feature.user.pago.model

import com.gert.tfgdam.feature.admin.lineaventa.model.LineaVenta
import com.gert.tfgdam.feature.admin.venta.model.Venta

data class FinalizarCompra(
    val venta: Venta,
    val lineasVenta: List<LineaVenta>
)